### ISSUE TITLE
Monitor conversation event adaptation failed

### ISSUE THREAD

**Issue by swatch17** [mitel-networks/micc-api-samples#8]
Hello
   I found that I couldn't listen to the session adaptation after I logged in to employeeInbox
in `connectToEmployeeHub  ` method
  `function connectToEmployeeHub(data) {
    var miccSignalr = new MiccSignalR(data.miccServer, data.access_token);
    miccSignalr.start(function () {
        miccSignalr.addSelfMonitor();
        miccSignalr.employeeConversationChanged(onEmployeeConversationChanged);
        miccSignalr.employeeConversationRemoved(onEmployeeConversationRemoved);
    });  
}`

**Comment by quinton-j** [comment]
I'm not sure what you mean by "couldn't listen to the session adaption".  Did you get an error?  Did no events trigger?  Were there any messages in the browser console logs?

**Comment by swatch17** [comment]
Hello
 I wan to know conversionId is it interactionId?

**Comment by shaapala** [comment]
Are you asking if conversationId that is used in the API calls and responses is the same as the interactionId used in web ignite UI?
Can you please give more details?
Thanks

**Comment by swatch17** [comment]
yes,in addition Can I listen to the tape if I use the conversatoinId?

**Comment by shaapala** [comment]
I don't understand what you mean by "can I listen to the tape"  What is this you are referring to?

**Comment by swatch17** [comment]
I would like to know what parameter data can be used to link to the call recording, or how to obtain the call recording based on the data

**Comment by shaapala** [comment]
@swatch17 what are you using for your call recording solution? MIR or the basic call recording?
